tests/Regression/Endpoint/UserEndpointTests.cs

```diff
@@ -0,0 +1,212 @@
+﻿using E.Deezer.Api;
+using E.Deezer.Endpoint;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace E.Deezer.Tests.Regression.Endpoint
+{
+    [TestFixture]
+    class UserEndpointTests
+    {
+        private static IUserEndpoint _user;
+
+        [OneTimeSetUp]
+        public void OneTimeSetUp()
+        {
+            var deezer = DeezerSession.CreateNew();
+            string token = "";
```

Comment:
Currently this is where a valid API token should be added before running the `Regression` testst.